src/views/SearchPageView.vue

```diff
@@ -141,6 +141,7 @@ export default {
               <th>Gene</th>
               <th>Disease</th>
               <th>Allelic Requirement</th>
+              <th>Molecular Mechanism</th>
```

Comment:
In all tables, we are displaying the header as "Mechanism" so for consistency, better to keep display it as "Mechanism" for now